### DISCUSSION TITLE
Feedback : Role editor

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#4018] [Q&A]
Hi, 
Leaving an editor member to delete a version of a database is quite dangerous (image below). I would want that this member just edit the project from his device that's all. Just a feedback ! :-)
![image](https://user-images.githubusercontent.com/7015197/223812097-f8e959f8-2bd2-496e-a7f6-df04775e3a51.png)

Thanks for all

**Comment by suricactus** [comment]
Thanks a lot for your great observation! Will be fixed soon!

**Reply by GlaDal** [reply]
You're welcome ! :-)
